File: C/Retangulo.c
#include <stdio.h>
#include <stdlib.h>

float calculateArea(float, float);
float calculatePerimeter(float, float);

int main() {
  float width, height;
  printf("Hello!\n"); 
  printf("Enter the width value: ");
  scanf("%f", &width);
  printf("Enter the height value: ");
  scanf("%f", &height);
  printf("The width is %f and the height is %f \n", width, height);
  printf("Let's make some calculations :P\n");
  printf("%f The area is: ", calculateArea(width, height));
  printf("%f The perimeter is: ", calculatePerimeter(width, height));
}

float calculateArea(float width, float height) {
  return width * height;
}

float calculatePerimeter(float width, float height) {
  return width * 2 + height * 2;
}

File: Java/Retangulo.java
public class Retangulo {
  public float width;
  public float height;

  public float getWidth() {
    return width;
  }

  public void setWidth(float width) {
    this.width = width;
  }

  public float getHeight() {
    return height;
  }

  public void setHeight(float height) {
    this.height = height;
  } 

  public Retangulo(float width, float height) {
    this.width = width;
    this.height = height;
  }

  public Retangulo() {
    
  }

  public float calculateArea(float width, float height) {
    return width * height;
  }

  public float calculatePerimeter(float width, float height) {
    return width * 2 + height * 2;
  }
  
}